shared-doc/build-and-deploy-the-quickstart.adoc

```diff
@@ -70,13 +70,22 @@ endif::mavenCommand[]
 . Make sure you xref:start_the_eap_standalone_server[start the {productName} server] as described above.
 . Open a terminal and navigate to the root directory of this quickstart.
 ifdef::reactive-messaging[]
-. Run this command to enable the MicroProfile Reactive Messaging functionality on the server
+. Run this command to enable the MicroProfile Reactive Messaging functionality on the server:
 +
 [source,subs="attributes+",options="nowrap"]
 ----
 $ __{jbossHomeName}__/bin/jboss-cli.sh --connect --file=enable-reactive-messaging.cli
 ----
 endif::reactive-messaging[]
+ifdef::microprofile-lra[]
```

Comment:
Please move this kind of content to the qs README.adoc, (re)using the common style and sections as seen starting  at https://github.com/wildfly/quickstart/blob/main/ejb-security-context-propagation/README.adoc?plain=1#L80C1 , and don't forget the config restoring, as seen starting at https://github.com/wildfly/quickstart/blob/main/ejb-security-context-propagation/README.adoc?plain=1#L278